quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -323,37 +322,77 @@ private static void addOption(CommandSpec spec, String command, boolean includeB
         List<PropertyMapper> mappers = new ArrayList<>(PropertyMappers.getRuntimeMappers());
 
         if (includeBuildTime) {
-            mappers.addAll(PropertyMappers.getBuiltTimeMappers());
+            mappers.addAll(PropertyMappers.getBuildTimeMappers());
         }
 
-        for (PropertyMapper mapper : mappers) {
-            String name = ARG_PREFIX + PropertyMappers.toCLIFormat(mapper.getFrom()).substring(3);
-            String description = mapper.getDescription();
-
-            if (description == null || commandSpec.optionsMap().containsKey(name)
-                || name.endsWith(ARG_PART_SEPARATOR)) {
-                continue;
+        if (includeBuildTime) {
```

Comment:
@pedroigor First of all thanks, forgotten these ones as they're not fetched by kc/quarkus config! :) 

Just a small change request: The same if condition in line 324 and line 328 could be merged. Also, for readability reasons I think it's better to create a method for the generation of the Features. Like -> 
```
private static void addOption(CommandSpec spec, String command, boolean includeBuildTime) {
        CommandSpec commandSpec = spec.subcommands().get(command).getCommandSpec();
        List<PropertyMapper> mappers = new ArrayList<>(PropertyMappers.getRuntimeMappers());

        if (includeBuildTime) {
            mappers.addAll(PropertyMappers.getBuildTimeMappers());
            addFeatureOptionsInArgGroup(commandSpec);
        }

        addMappedOptionsToArgGroups(commandSpec, mappers);
    }

    private static void addFeatureOptionsInArgGroup(CommandSpec commandSpec) { 
    // logic from PR stays here
    }
    ```

Reply:
Right. Ideally, I think we should have something in our mappers to allow adding these "dynamic" options. 

We should come back to this part again to review this as well as avoid any unnecessary processing when starting the server in production (e.g.: kc.sh start, with no option). For instance, we don't need to initialize mappers at all when starting the server.